solr/core/src/test/org/apache/solr/handler/admin/LoggingHandlerTest.java

```diff
@@ -66,27 +70,19 @@ public void testLogLevelHandlerOutput() throws Exception {
                  config.getLoggerConfig(PARENT_LOGGER_NAME));
     assertEquals(Level.DEBUG, config.getLoggerConfig(CLASS_LOGGER_NAME).getLevel());
 
-    assertQ("Show Log Levels OK",
-            req(CommonParams.QT,"/admin/logging")
-            ,"//arr[@name='loggers']/lst/str[.='"+CLASS_LOGGER_NAME+"']/../str[@name='level'][.='DEBUG']"
-            ,"//arr[@name='loggers']/lst/str[.='"+PARENT_LOGGER_NAME+"']/../null[@name='level']"
-            );
+    SolrClient client = new EmbeddedSolrServer(h.getCore());
+    ModifiableSolrParams mparams = new ModifiableSolrParams();
+    mparams.set("set", PARENT_LOGGER_NAME + ":TRACE");
+    client.request(new GenericSolrRequest(SolrRequest.METHOD.GET, "/admin/info/logging", mparams));
 
-    assertQ("Set a (new) level",
```

Comment:
@dsmiley not sure if we still need to test "Set a (new) level" or "Show Log Levels OK" ?

Reply:
btw I run all tests and all they pass.

Reply:
AFAICT, the purpose of these assertions is to assert the response of getting the log levels from Solr.  Your changes removed that; thus it's untested (right?).  It should be tested to protect against regressions in the format or what levels are included.